comps/llms/README.md

```diff
@@ -259,6 +314,46 @@ curl http://${your_ip}:9000/v1/health_check\
   -H 'Content-Type: application/json'
 ```
 
+### 3.2 Verify the LLM Service
+
+#### 3.2.1 Verify the TGI Service
+
+```bash
+curl http://${your_ip}:8008/generate \
+  -X POST \
+  -d '{"inputs":"What is Deep Learning?","parameters":{"max_new_tokens":17, "do_sample": true}}' \
+  -H 'Content-Type: application/json'
+```
+
+#### 3.2.2 Verify the vLLM Service
+
+```bash
+curl http://${your_ip}:8008/v1/completions \
+  -H "Content-Type: application/json" \
+  -d '{
+  "model": ${your_hf_llm_model},
+  "prompt": "What is Deep Learning?",
+  "max_tokens": 32,
+  "temperature": 0
+  }'
+```
+
+#### 3.2.3 Verify the Ray Service
+
+```bash
+curl http://${your_ip}:8008/v1/chat/completions \
+  -H "Content-Type: application/json" \
+  -d '{
+  "model": ${your_hf_llm_model},
+  "messages": [
+        {"role": "assistant", "content": "You are a helpful assistant."},
+        {"role": "user", "content": "What is Deep Learning?"},
+    ],
+  "max_tokens": 32,
+  "stream": True
+  }'
+```
+
 ### 3.2 Consume LLM Service
 
 You can set the following model parameters according to your actual needs, such as `max_new_tokens`, `streaming`.
```

Comment:
There are a couple more long lines in the section below that you could break to make shorter (the -d query lines)